### ISSUE TITLE
Please add redirects from https://server.tld/user to https://server.tld/@user

### ISSUE THREAD

**Issue by jeroenpraat** [tootsuite/mastodon#13550]
### Pitch and Motivation

People (even me sometimes) forget the @ symbol in https://server.tld/@user, especially because some well known site do it without the @. Actually I like the @ there, but it would be a big improvement to just make an auto-redirect, when people forget the @. I think it's a minor change, with big positive consequences.

**Comment by mstdn** [comment]
That could be but that way we must reserve certain usernames. For example @admin. Otherwise you will be redirected.

**Comment by Todon-nl** [comment]
> That could be but that way we must reserve certain usernames. For example @admin. Otherwise you will be redirected.

Yes that's true, but that isn't a problem of course. This is feature request for non-admin users. Also other links would have priority, like 'settings'

**Comment by koyuawsmbrtn** [comment]
Could be easily solved with redirects. Just configure your webserver to redirect from `https://server.tld/@*` to `https://server.tld/@$1`

**Comment by jeroenpraat** [comment]
@koyuawsmbrtn No that gives problem with used names, like admin, settings, about, auth. etc. I think this must be in the code itself.

**Comment by koyuawsmbrtn** [comment]
Oh well, true
